lib/ddtrace/contrib/rails/integration.rb

```diff
@@ -16,7 +16,9 @@ class Integration
         register_as :rails, auto_patch: false
 
         def self.version
-          Gem.loaded_specs['rails'] && Gem.loaded_specs['rails'].version
+          # Check if Railties are present as well...
+          (Gem.loaded_specs['rails'] && Gem.loaded_specs['rails'].version) \
+          || (Gem.loaded_specs['railties'] && Gem.loaded_specs['railties'].version)
```

Comment:
I believe we can simplify this to only check for the `'railties'` gem, as every version of Rails we support has an empty top-level `rails` gem, and `::Rails` is defined by the `railties` gem (which always matches the same version of the `rails` meta gem).

Reply:
Hmmm, very true. I like this... will make this change.